AdventOfCode2021/day5/day5_1.py

```diff
@@ -1,37 +1,45 @@
 #!python3
 
 from line import Line
+from point import Point
 from typing import Dict, List, Tuple
-import os
 
 
 def file_read(file_name) -> List[Line]:
     lines: List[Line] = []
     with open(file_name) as f:
         for line in f:
-            points = line.strip().split(" -> ")
-            l: Line = Line(points[0], points[1])
+            point = line.strip().split(" -> ")
+            start_point = list(map(int, point[0].split(",")))
+            end_point = list(map(int, point[1].split(",")))
+            p_start: Point = Point(start_point[0], start_point[1])
+            p_end: Point = Point(end_point[0], end_point[1])
+            l: Line = Line(p_start, p_end)
             if (l.horizontal() or l.vertical()):
                 lines.append(l)
     return lines
 
 
-def points_number_of_two_overlaps(lines: List[Line]) -> int:
-    point_overlaps_number: Dict[Tuple[int, int], int] = {}
+def point_overlaps(lines: List[Line]) -> Dict[Point, int]:
```

Comment:
By convention, functions names are verbs, so `get_point_overlaps` could be a better name.

Reply:
got it